packages/manager/src/views/DashboardPage/components/Analytics/components/LineChart/LineChart.tsx

```diff
@@ -0,0 +1,124 @@
+import React from 'react';
+import { useGetMonthlyDeploymentChart } from '@app/services/analytics';
```

Comment:
Kindly keep all the api calls from component towards pages. That is the parent component in views. 

Keep these components as dumb as possible.